frontend/views/pages/Join.vue

```diff
@@ -149,7 +137,13 @@ export default ({
           creatorPicture,
           message
         }
-        this.pageStatus = 'SIGNING'
+
+        if (!syncFinished) {
+          this.pageStatus = 'SIGNING'
+          sbp('okTurtles.events/once', LOGIN, () => this.accept())
+        } else {
+          this.accept()
+        }
```

Comment:
`accept` function should be called once only when the validation is passed and also the user finishes logging in.

Reply:
See my comment above: https://github.com/okTurtles/group-income/pull/1603#discussion_r1206179653